package.json

```diff
@@ -6,6 +6,9 @@
   "bin": {
     "addons-linter": "bin/addons-linter"
   },
+  "engines": {
+    "node": ">=10.15"
```

Comment:
The specific version feels rather random, afaik 10.x is LTS as of 10.12 - why did we pick that particular version?

Reply:
There was a reason, but I can't remember it. We can lower it to 10.13 since that was the first real LTS release.

Reply:
That'd be good, thanks!